scanner/mappers/mappers.go

```diff
@@ -615,16 +622,21 @@ func severityAndScores(vuln *claircore.Vulnerability, nvdScores map[string]nvdsc
 	default:
 		// Everything else uses NVD.
 		values.severity = vuln.Severity
-		if nvd, ok := nvdScores[vuln.ID]; ok {
-			switch {
-			case strings.HasPrefix(nvd.Version, "2"):
-				values.v2Score = float32(nvd.BaseScore)
-				values.v2Vector = nvd.VectorString
-			case strings.HasPrefix(nvd.Version, "3"):
-				values.v3Score = float32(nvd.BaseScore)
-				values.v3Vector = nvd.VectorString
-			default:
-				errList.AddError(fmt.Errorf("unsupported CVSS score version: %s", nvd.Version))
+		if nvd, ok := nvdVulns[vuln.ID]; ok {
+			if len(nvd.Metrics.CvssMetricV31) > 0 {
+				cvssv31 := nvd.Metrics.CvssMetricV31[0]
```

Comment:
getting this one is likely correct, but technically we want the one with `Type == "Primary"`

Reply:
See the enricher.